09week/hackernews/script.js

```diff
@@ -1 +1,59 @@
 'use strict';
+
+class Hacker extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			list: []
+		}
+	}
+
+	componentWillMount(){
+		// store state changes
+		const stories = []
+		// create list 
+		for (let i=1; i<11; i++) {
+			const url = `https://hacker-news.firebaseio.com/v0/item/${i}.json?print=pretty `; // Hacker News API
```

Comment:
You are using the top 10 articles instead of using ```https://hacker-news.firebaseio.com/v0/topstories.json?print=pretty``` to fetch the list of articles and displaying those.